packages/svelte/src/compiler/validate-options.js

```diff
@@ -83,6 +83,8 @@ export const validate_component_options =
 
 			namespace: list(['html', 'svg', 'foreign']),
 
+			modernAst: boolean(false),
```

Comment:
This is the only change necessary to fix this. The other changes are unnecessary as they're for a different function, the `parse` function. Can you revert those other changes?

Reply:
Woops, reverted.